daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/VariableMap1.scala

```diff
@@ -222,7 +300,7 @@ class VariableMap private(vTable: Map[GlobalQName, MStackOf[VariableInstance]])
   /**
    * For testing mostly.
    */
-  def getVariableBindings(qn: GlobalQName): MStackOf[VariableInstance] = {
+  def getVariableBindings(qn: GlobalQName): Queue[VariableInstance] = {
```

Comment:
We used to use a stack data structure. Why is this Queue and not Stack? It would seem the most natural replacement for MStack would be Stack. 

This would eliminate a bunch of confusion below where we get a "queue" from the vtable using a "qname", i.e., where "Q" gets overloaded about Qualified (as in names) and Queue (as in data structure). 

Can we use Stack?

Reply:
So mutable Stack is deprecated in Scala 2.12.  It's back in 2.13, but I believe we are still primarily targetting 2.12

Reply:
Well, a cheap collection class is ArrayBuffer, if we don't care about it really being stack-like or queue-like. I guess I don't like using a Queue as a bag if mutable set, or just array buffer is sufficient. What is the access pattern needed here? It used to be MStack, so I'm surprised Queue works.

Reply:
A stack is the ideal structure for this as with newVariableInstance we are pushing and popping VariableInstance's as they go in and out of scope.  That being said, you are right that an ArrayBuffer will work just fine and I'm sure would be more efficient. I'll make the change.